app_s2i/templates/includes/_jumbotron.html

```diff
@@ -1,6 +1,6 @@
 <div class="container">
   <div style="background:transparent !important" class="jumbotron jumbotron-fluid">
-    <h1 class="display-4">Welcome to the DevOps Learning Community</h1>
+    <h1 class="display-5">Welcome to the DevOps Learning Community</h1>
```

Comment:
The home page doesn't look good with this change. If you insist on adding it, verify it is applicable on all the rest, but the home page.

Reply:
rolled back